File: TodoApp/Views/AppLogo.cs
namespace TodoApp.Views {
    using DevExpress.XtraEditors;
    using TodoApp.ViewModels;

    public partial class AppLogo : XtraUserControl {
        public AppLogo() {
            InitializeComponent();
            // Initializing bindings only at runtime   
            if(!mvvmContext.IsDesignMode)
                InitializeBindings();
        }
        void InitializeBindings() {
            // Initialize the Fluent API
            var fluent = mvvmContext.OfType<AppLogoViewModel>();
            // Bind the Greeting property to the label Text
            fluent.SetBinding(greetLabel, lbl => lbl.Text, x => x.Greeting);
        }
    }
}

File: VB/Views/AppLogo.vb
Imports DevExpress.XtraEditors
Imports TodoApp.ViewModels

Namespace TodoApp.Views
    Public Partial Class AppLogo
        Inherits XtraUserControl
        Public Sub New()
            InitializeComponent()
            ' Initializing bindings only at runtime   
            If Not mvvmContext.IsDesignMode Then InitializeBindings()
        End Sub
        Sub InitializeBindings()
            ' Initialize the Fluent API
            Dim fluent = mvvmContext.OfType(Of AppLogoViewModel)()
            ' Bind the Greeting property to the label Text
            fluent.SetBinding(greetLabel, Function(lbl) lbl.Text, Function(x) x.Greeting)
        End Sub
    End Class
End Namespace
